dataland-frontend/src/components/forms/parts/elements/basic/ExtendedDataPointFormField.vue

```diff
@@ -199,10 +192,6 @@ export default defineComponent({
   },
   props: {
     ...FormFieldPropsWithPlaceholder,
-    checkValueValidity: {
-      type: Function as unknown as () => (dataPoint: unknown) => boolean,
-      default: (): boolean => false,
-    },
```

Comment:
Die prop, die du hier gelöscht hast, wird aktuell im Frontend-src-code bei insgesamt 4 Nutzungen der Component `ExtendedDataPointFormField` noch mit einem Wert versehen.

Reply:
(Da die prop jedoch immer mit der Funktion `hasDataPointProperValue` befüllt ist, kann man die prop auch entfernen und die Funktion selbst direkt in dieser Vue-component hier importieren und nutzen.)

Reply:
Alle Nutzungen der Props wurden entfernt.